.github/workflows/test.yml

```
@@ -0,0 +1,160 @@
+name: Test Updating Version
+
+on:
+  worfklow_dispatch: {}
```

actionlint (1.7.7)

unknown Webhook event "worfklow_dispatch". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)